src/components/SingleRev.jsx

```diff
@@ -43,7 +45,9 @@ export const SingleRev = ({ loading, setLoading }) => {
           <p>Category: {oneRev.category}</p>
           <p>Designer: {oneRev.designer}</p>
           <p>Posted at: {oneRev.created_at}</p>
-          <button onClick={handleVote}> 👍 {oneRev.votes}</button>
+          <p>Votes: {oneRev.votes}</p>
+          <button onClick={handleVoteInc}> 👍 </button> &nbsp;
+          <button onClick={handleVoteInc}> 👎 </button>
```

Comment:
ideally you wouldn't add this now as it's for the next ticket, but don't worry about it for this one.